hapi/build.gradle.kts

```diff
@@ -34,7 +34,7 @@ tasks.cloneHederaProtobufs {
     // uncomment below to use a specific tag
     // tag = "v0.53.0"
     // uncomment below to use a specific branch
-    branch = "add-throttle-at-consensus"
```

Comment:
this should never have made it to develop but I'm adding a note here so mine doesn't get merged until I update back to main.